### PR TITLE
MINIFICPP-2080 Add missing processors to README.md

### PR DESCRIPTION
**Pull request by lordgamez**
https://issues.apache.org/jira/browse/MINIFICPP-2080

------------
Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [ ] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI results for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Review by martinzink — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -63,45 +63,45 @@ MiNiFi - C++ supports the following C++ processors:
...
+| MQTT                             | [ConsumeMQTT](PROCESSORS.md#consumeMQTT)<br/>[PublishMQTT](PROCESSORS.md#publishMQTT)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | -DENABLE_MQTT=ON             |
```
Comment (fgerlits) [inline README.md:89]: the links should be all lowercase:
```suggestion
| MQTT                             | [ConsumeMQTT](PROCESSORS.md#consumemqtt)<br/>[PublishMQTT](PROCESSORS.md#publishmqtt)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | -DENABLE_MQTT=ON             |
```

File: README.md [hunk 2]
```diff
@@ -63,45 +63,45 @@ MiNiFi - C++ supports the following C++ processors:
...
+| Azure                            | [AzureStorageCredentialsService](CONTROLLERS.md#azurestoragecredentialsservice)<br/>[PutAzureBlobStorage](PROCESSORS.md#putazureblobatorage)<br/>[DeleteAzureBlobStorage](PROCESSORS.md#deleteazureblobstorage)<br/>[FetchAzureBlobStorage](PROCESSORS.md#fetchazureblobstorage)<br/>[ListAzureBlobStorage](PROCESSORS.md#listazureblobstorage)<br/>[PutAzureDataLakeStorage](PROCESSORS.md#putazuredatalakestorage)<br/>[DeleteAzureDataLakeStorage](PROCESSORS.md#deleteazuredatalakestorage)<br/>[FetchAzureDataLakeStorage](PROCESSORS.md#fetchazuredatalakestorage)<br/>[ListAzureDataLakeStorage](PROCESSORS.md#listazuredatalakestorage) | -DENABLE_AZURE=ON            |
```
Comment (fgerlits) [inline README.md:79]: typo: 'a' -> 's'
```suggestion
| Azure                            | [AzureStorageCredentialsService](CONTROLLERS.md#azurestoragecredentialsservice)<br/>[PutAzureBlobStorage](PROCESSORS.md#putazureblobstorage)<br/>[DeleteAzureBlobStorage](PROCESSORS.md#deleteazureblobstorage)<br/>[FetchAzureBlobStorage](PROCESSORS.md#fetchazureblobstorage)<br/>[ListAzureBlobStorage](PROCESSORS.md#listazureblobstorage)<br/>[PutAzureDataLakeStorage](PROCESSORS.md#putazuredatalakestorage)<br/>[DeleteAzureDataLakeStorage](PROCESSORS.md#deleteazuredatalakestorage)<br/>[FetchAzureDataLakeStorage](PROCESSORS.md#fetchazuredatalakestorage)<br/>[ListAzureDataLakeStorage](PROCESSORS.md#listazuredatalakestorage) | -DENABLE_AZURE=ON            |
```

File: README.md [hunk 3]
```diff
@@ -63,45 +63,45 @@ MiNiFi - C++ supports the following C++ processors:
...
+| OpenCV                           | [CaptureRTSPFrame](PROCESSORS.md#captureRTSPFrame)<br/>[MotionDetector](PROCESSORS.md##motiondetector)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | -DENABLE_OPENCV=ON           |
```
Comment (fgerlits) [inline README.md:91]: two typos: `##` -> `#`
```suggestion
| OPC                              | [FetchOPCProcessor](PROCESSORS.md#fetchopcprocessor)<br/>[PutOPCProcessor](PROCESSORS.md#putopcprocessor)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | -DENABLE_OPC=ON              |
| OpenCV                           | [CaptureRTSPFrame](PROCESSORS.md#captureRTSPFrame)<br/>[MotionDetector](PROCESSORS.md#motiondetector)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | -DENABLE_OPENCV=ON           |
```

Comment (fgerlits) [inline README.md:91]: should be all lowercase:
```suggestion
| OPC                              | [FetchOPCProcessor](PROCESSORS.md#fetchopcprocessor)<br/>[PutOPCProcessor](PROCESSORS.md##putopcprocessor)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | -DENABLE_OPC=ON              |
| OpenCV                           | [CaptureRTSPFrame](PROCESSORS.md#capturertspframe)<br/>[MotionDetector](PROCESSORS.md##motiondetector)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | -DENABLE_OPENCV=ON           |
```

File: README.md (hunk 1 above)
Comment (lordgamez) [reply README.md]: Updated in b1ad02d8801340f3803cda61a6e5378d5c37d8b4

File: README.md (hunk 2 above)
Comment (lordgamez) [reply README.md]: Updated in b1ad02d8801340f3803cda61a6e5378d5c37d8b4

File: README.md (hunk 3 above)
Comment (lordgamez) [reply README.md]: Updated in b1ad02d8801340f3803cda61a6e5378d5c37d8b4

**Review by fgerlits — APPROVED** [review]

**Review by adamdebreceni — APPROVED** [review]

**Pull request closed without merging**